frontend/src/__tests__/integration/pages/projects/ProjectView.spec.ts

```diff
@@ -26,6 +24,9 @@ test('Project view page', async ({ page }) => {
   await expect(page.getByText('Non-DS Project 1', { exact: true })).toBeVisible();
   await expect(page.getByText('Non-DS Project 2', { exact: true })).toBeVisible();
   await expect(page.getByText('Non-DS Project 3', { exact: true })).toBeVisible();
+
+  await page.locator('#search-field-toggle').click();
+  await expect(page.getByRole('listitem')).toHaveCount(2);
```

Comment:
Uh, meant both check 0 & 1 as intended values but ALSO check the length -- and not just trust they are first and last is what we want... that seems like a half-test. Like this is currently.